### ISSUE TITLE
Debug about Mobs and Life

### ISSUE THREAD

**Issue by LightFire53** [ParadiseSS13/Paradise#5822]
Well. This was a thing that happened. Posting it because it may or may not be an issue of some kind.

ADMIN LOG: Process 'mob' is hung and will be restarted.

DEBUG: mob process hung at tick #1052. Process was unresponsive for 90.0551 seconds and was restarted. Last task: 0. Last Object Type: /mob/living/carbon/human

ADMIN LOG: mob process hung at tick #1052. Process was unresponsive for 90.0551 seconds and was restarted. Last task: 0. Last Object Type: /mob/living/carbon/human

DEBUG: Runtime in code/controllers/ProcessScheduler/core/process.dm,202: mob process hung at tick #1052. Process was unresponsive for 90.0551 seconds and was restarted. Last task: 0. Last 
[07:27:53] Runtime in code/controllers/ProcessScheduler/core/process.dm,202: mob process hung at tick #1052. Process was unresponsive for 90.0551 seconds and was restarted. Last task: 0. Last Object Type: /mob/living/carbon/human
   src: mob (/datum/controller/process/mob)
[07:27:53] Runtime in life.dm,29: Cannot read null.current_cycle
   Caught by process: mob
   src: The grey adult slime (76) () (/mob/living/carbon/slime)
   src.loc: The reinforced floor (173,82,1) (/turf/simulated/floor/engine)
[07:27:53] Runtime in life.dm,234: Cannot read null.current_cycle
   Caught by process: mob
   src: Clown Soldier () (/mob/living/carbon/human/human)
   src.loc: The floor (45,37,6) (/turf/simulated/floor/plasteel/airless)
[07:27:53] Runtime in life.dm,1030: Cannot read null.current_cycle
   Caught by process: mob
   src: Bonk Consortium Operative #1 (as Alex Rockwall) () (/mob/living/carbon/human)
   src.loc: space (38,125,1) (/turf/space)
[07:27:59] Runtime in life.dm,1030: Cannot read null.current_cycle
   Caught by process: mob
   src: Bonk Consortium Operative #1 (as Alex Rockwall) () (/mob/living/carbon/human)
   src.loc: space (38,125,1) (/turf/space)
   (This error will now be silenced for 10 minutes)

**Comment by LightFire53** [comment]
It should be noted that immediately after these logs popped, we started getting ahelps involving death, or being both living and dead. For example, an IPC with over 190 brute damage could still attempt to talk, in addition to their screen being perfectly fine, when they really were dead.

**Comment by Fox-McCloud** [comment]
What the. How in the world did a slime end up processing reagents to begin with?

**Comment by GelatinousGlob** [comment]
If this happened in the round I was on it was nukies, I got shot in the chest and had shrapnel inside but could walk around and talk and my heartbeat was green. I scanned myself later with a scanner after getting a lot of medicine and I had 160 brute damage. I was slow but otherwise fine.

Also the combat shotgun would recock itself (unless this is a new feature?) and in curiously I shot a nuke ops with 7 pulse shotguns lugs and he could still talk and was fine while being beat by greytide.
